### ISSUE TITLE
What's the function of the parameter "--save_sample_prompt" in "launch.sh" file ?

### ISSUE THREAD

**Issue by zakinp** [ShivamShrirao/diffusers#63]
the updated  dreambooth can train multiple concepts at once. But i don't understand the function of the parameter "--save_sample_prompt" in "launch.sh"

**Comment by geocine** [comment]
`--save_sample_prompt` is the prompt used to generate `save_samples` at ever n steps using `save_interval` . The `save_interval` will only get triggered if it is less than `max_training_steps`. By default all of these new parameters are turned off by default due to the default values set. Meaning checkpoints won't be saved at every N interval and a sample won't be created for them. This was a change brought about by https://github.com/ShivamShrirao/diffusers/issues/58

**Comment by 1blackbar** [comment]
Wouldbe nice if we could specify multiple prompts separated by / maybe or *  , to test various styles not just one

**Comment by zakinp** [comment]
> `--save_sample_prompt` is the prompt used to generate `save_samples` at ever n steps using `save_interval` . The `save_interval` will only get triggered if it is less than `max_training_steps`. By default all of these new parameters are turned off by default due to the default values set. Meaning checkpoints won't be saved at every N interval and a sample won't be created for them. This was a change brought about by #58

thanks, it can generate a preview of the specific prompt every n steps

**Comment by zakinp** [comment]
> Wouldbe nice if we could specify multiple prompts separated by / maybe or * , to test various styles not just one

yeah, the current function of this parameter can only generate one trained concept in every n steps. i don't know there is any way to check all the concepts trained in model.
